app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -5,13 +5,13 @@ package com.github.se.cyrcle.model.user
  *
  * @property userId The unique identifier of the user.
  * @property username The username of the user.
- * @property profilePictureUrl The path to the profile picture of the user on the cloud storage.
- *   (Not implemented)
+ * @property profilePictureCloudPath The path to the profile picture of the user on the cloud
+ *   storage. (Not implemented)
```

Comment:
Even though it is not this PR, having `Not implemented` in the documentation would be bad practice. Having dependency in the documentation (because that's what we're gonna have to do once Cloud storage is up, will be to update said documentation. If a system is not yet implemented, only this system should be marked as such. If I'm mistaken and this classes are said system, then it would be better to have it written in big inside the description, not the property/param.